lib/Extractor/Candidates.cpp

```diff
@@ -35,11 +35,16 @@
 #include <sstream>
 #include <unordered_set>
 #include <tuple>
+#include "llvm/Analysis/ValueTracking.h"
 
 static llvm::cl::opt<bool> ExploitBPCs(
     "souper-exploit-blockpcs",
     llvm::cl::desc("Exploit block path conditions (default=true)"),
     llvm::cl::init(true));
+static llvm::cl::opt<bool> HarvestKnownBits(
+    "souper-known-bits-analysis",
+    llvm::cl::desc("Perform known bits analysis (default = false)"),
```

Comment:
please make the variable name and the command line option match up a bit better. so for example if the variable is HarvestKnownBits then make the option souper-harvest-known-bits